pkg/agent/openflow/pipeline.go

```diff
@@ -898,6 +947,43 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 		Done()
 }
 
+// ipv6Flows generates the flows to allow IPv6 packets from link-local addresses and
+// handle multicast packets, ND solicitation and ND advertisement packets properly.
+func (c *client) ipv6Flows(category cookie.Category) []binding.Flow {
+	var flows []binding.Flow
+	// TODO: Remove the flag after finishing Antrea Proxy changes
+	if !c.enableProxy {
+		_, ipv6LinkLocalIpnet, _ := net.ParseCIDR(ipv6LinkLocalAddr)
+		_, ipv6MulticastIpnet, _ := net.ParseCIDR(ipv6MulticastAddr)
+		flows = append(flows,
+			// Allow IPv6 packets from link-local addresses.
+			c.pipeline[spoofGuardTable].BuildFlow(priorityNormal).MatchProtocol(binding.ProtocolIPv6).
```

Comment:
Could we add comments before flows to explain why there are needed?

Reply:
Sure, will add more comments here.

Reply:
Done.